SharedPackages/UIComponents/Package.swift

```diff
@@ -0,0 +1,52 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+//
+//  Package.swift
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import PackageDescription
+
+let package = Package(
+    name: "UIComponents",
+    platforms: [
+        .iOS("15.0"),
+        .macOS("11.4")
+    ],
+    products: [
+        .library(
+            name: "UIComponents",
+            targets: ["UIComponents"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/duckduckgo/DesignResourcesKit", exact: "4.2.0")
+    ],
+    targets: [
+        .target(
+            name: "UIComponents",
+            dependencies: [
+                "DesignResourcesKit",
+            ],
+            resources: [
+                .process("Resources/Assets.xcassets")
```

Comment:
The new package technically doesn't have an Asset Catalog, so this shows as a (harmless) warning in Xcode. We can either remove this, or add an empty asset catalog - either way is fine by me. :)

Reply:
Interesting. I created the asset catalog, but when it’s empty, it's of a bunch of empty folders that Xcode doesn’t like. I added a SampleColor just so it doesn’t complain. I’ll let the team know that this can be removed, it’s only there to facilitate having an asset catalog ready to go